spec/models/workflow_spec.rb

```diff
@@ -15,6 +15,8 @@
     let(:translatable) { create(:workflow_with_contents, build_extra_contents: true) }
     let(:translatable_without_content) { build(:workflow, build_contents: false) }
     let(:primary_language_factory) { :workflow }
+    let(:private_project) { create(:project, private: true) }
```

Comment:
That's where it's used. Here's where it's built:
https://github.com/zwolf/Panoptes/blob/f220f2000a67f806d040e83eb92c43a1d421d5ff/spec/models/workflow_spec.rb#L18

Unless you mean that there's another way to ensure the translatable support has everything it needs?